make-release.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+if [ -z "$1" ]; then
+    echo "Usage: $0 <release_version>"
+    exit 1
+fi
+
+RELEASE_VERSION=$1
+
+# Update CHANGES.md
+sed -i "s/## Unreleased/&\n\n## $RELEASE_VERSION\n\n%RELEASE NOTES TEMPLATE - DON'T EDIT THE LINES STARTING WITH % \n%This description should answer the \"who needs this update?\" question.\n/" CHANGES.md
+${EDITOR:-vim} +9 CHANGES.md
+sed -i '/^%/d' CHANGES.md
```

Comment:
Why can't they edit lines with `%`? Wouldn't removing them simply make the later sed a no-op?